### PR TITLE
Update participate.html with element invite link.

### PR DESCRIPTION
**Pull request by magmaxt**

### CODE REVIEW

File: participate.html [hunk 1]
```diff
@@ -46,6 +46,13 @@ <h2>Participate!</h2>
                 know the answers.
               </li>
 
+	      <li>
+                Join our deal.II space 
+		<a href="https://matrix.to/#/#dealii:matrix.org">#dealii:matrix.org</a> at deal.II. 
+		There is a general chat for all	deal.II related questions and a lounge
+		for not so on-topic chat.
```
Comment (peterrum) [inline participate.html:53]:
```suggestion
		for not so on-topic chats.
```

File: participate.html [hunk 2]
```diff
@@ -46,6 +46,13 @@ <h2>Participate!</h2>
                 know the answers.
               </li>
 
+	      <li>
+                Join our deal.II space
```
Comment (peterrum) [inline participate.html:50]:
```suggestion
                Join our deal.II element chat 
```

File: participate.html [hunk 3]
```diff
@@ -46,6 +46,13 @@ <h2>Participate!</h2>
                 know the answers.
               </li>
 
+	      <li>
+                Join our deal.II element chat 
+		<a href="https://matrix.to/#/#dealii:matrix.org">#dealii:matrix.org</a> at deal.II. 
+		There is a general chat for all	deal.II related questions and a lounge
+		for not so on-topic chats.
```
Comment (tjhei) [inline participate.html:53]:
```suggestion
		at <a href="https://matrix.to/#/#dealii:matrix.org">#dealii:matrix.org</a>. 
		There you will find the room general chat for all deal.II related questions and a lounge
		for not so on-topic conversations.
```

Comment (masterleinad) [inline participate.html]:
```suggestion
                Join our deal.II <a href="https://matrix.to/#/#dealii:matrix.org">element chat</a>.
```
should be sufficient.

Comment (magmaxt) [reply participate.html]: updated accordingly.

**Review by masterleinad — APPROVED** [review]

**Pull request merged**